### DISCUSSION TITLE
Somehow my generation time doubled

### DISCUSSION THREAD

**Discussion by patientx** [lshqqytiger/stable-diffusion-webui-directml#303] [Q&A]
While generating 512x768 images , I was getting around 2.3 sec / it with my 6600 and it is also the same with comfyui but over the last few days something happened and now I am getting around 4.8 sec / it now. What could be the problem , any ideas ?

Here is my launch parameters  : --opt-sub-quad-attention --no-half-vae --disable-nan-check --medvram
Here are the extensions I have currently installed : 

Stable-Diffusion-Webui-Civitai-Helper
sd-webui-agent-scheduler 
sd-webui-animatediff 
sd-webui-controlnet
sd-webui-reactor

All of them I had last week too.

**Comment by hookechoes** [comment]
Unfortunately having the same issue here.  Incredibly slow gens on this branch.

**Comment by Drakensangs** [comment]
Same here, generation time is definitely slower.

**Comment by ratunku** [comment]
As a workaround, what might help: try switching to another model after changing the resolution, generate one picture and then switch back to the indented model. The generation should be quicker now. Eventually there is also an option to reload the model upon every generation, maybe this might help too.

**Reply by patientx** [reply]
Unfortunately this doesn't change anything, the basic speed is low it doesn't matter which model to use or changing models. And I am actually using the refiner option that is intended for sdxl with sd1.5 models to get variety in results (for example using fantexi as initial model , that enabling refiner and using more realistic model in there with switching at 0.5 , we get realistic results but at the same time more rich content) and if your suggestion was to work properly I would see the second model render at twice the speed or at least faster but no matter still the same. Whether its one model or another or two models etc. Also not using any loras, and disabled almost all extensions that could effect the app.
